### PR TITLE
Various windows migration changes (mostly build files)

### PR DESCRIPTION
**Pull request by b-maldoca**

### CODE REVIEW

**Comment by b-maldoca** [comment]
@googlebot I signed it!

File: bazel/mini_chromium.BUILD [hunk 1]
```diff
@@ -12,12 +12,165 @@
...
 # limitations under the License.
 
-licenses(["notice"])  # Apache v2.0
+licenses(["notice"])
 
 package(
     default_visibility = ["//visibility:public"],
 )
 
+DEFAULT_MINI_CHROMIUM_WIN_COPTS = [
```
Comment (jennypullman) [inline bazel/mini_chromium.BUILD:21]: Would it make sense to pull some of the common copts into a common DEFAULT_BASE_COPTS? (i.e. any shared flags between DEFAULT_MINI_CHROMIUM_WIN_COPTS and DEFAULT_MINI_CHROMIUM_LINUX_COPTS)

File: bazel/variables.bzl [hunk 2]
```diff
@@ -1,4 +1,34 @@
-DEFAULT_COPTS = [
+DEFAULT_WIN_COPTS = [
```
Comment (jennypullman) [inline bazel/variables.bzl:1]: why is this in the build file as well as the bzl file?

File: clang-format.sh [hunk 3]
```diff
@@ -14,8 +14,6 @@
 
 # Runs clang-format and show diff.
 
-alias clang-format=clang-format-9
```
Comment (jennypullman) [inline clang-format.sh:17]: Why remove this?

File: maldoca/base/file.cc [hunk 4]
```diff
@@ -18,8 +18,9 @@
 
 #include "maldoca/base/file.h"
 
-// TODO(someone): fix for windows
```
Comment (jennypullman) [inline maldoca/base/file.cc:21]: Is there a github issue for this?

File: clang-format.sh (hunk 3 above)
Comment (b-maldoca) [reply clang-format.sh:17]: we've upgraded the clang version and the docker image only contains 1 clang-format version anyways. hence "clang-format" is sufficient.

File: maldoca/base/file.cc (hunk 4 above)
Comment (b-maldoca) [reply maldoca/base/file.cc:21]: no

File: bazel/variables.bzl (hunk 2 above)
Comment (b-maldoca) [reply bazel/variables.bzl:1]: DEFAULT_COPTS r build options for maldoca. while DEFAULT_MINI_CHROMIUM_WIN_COPTS r build options for windows.

i renamed DEFAULT_COPTS to DEFAULT_MALDOCA_COPTS to make it explicit.

File: bazel/mini_chromium.BUILD (hunk 1 above)
Comment (b-maldoca) [reply bazel/mini_chromium.BUILD]: done; created DEFAULT_MINI_CHROMIUM_BASE_COPTS

File: bazel/variables.bzl (hunk 2 above)
Comment (b-maldoca) [reply bazel/variables.bzl]: i would prefer keeping them separate (at least for now). however, followed your advice in option 2 and added comments to make it explicit.

File: maldoca/base/file.cc (hunk 4 above)
Comment (b-maldoca) [reply maldoca/base/file.cc:21]: i am a little confused. this todo is removed as it is fixed (i.e. the code is not required for chrome)?

Comment (jennypullman) [reply maldoca/base/file.cc:21]: Lol sorry, I was confused - I thought this was in green.

File: bazel/variables.bzl [hunk 5]
```diff
@@ -1,4 +1,37 @@
-DEFAULT_COPTS = [
+# Those are the build flags used to build MalDocA (maldoca/*).
+# The flags are not applied to build third party dependencies.
+
```
Comment (jennypullman) [inline bazel/variables.bzl:3]: nit: delete this empty line.

File: bazel/variables.bzl [hunk 6]
```diff
@@ -1,4 +1,37 @@
-DEFAULT_COPTS = [
+# Those are the build flags used to build MalDocA (maldoca/*).
+# The flags are not applied to build third party dependencies.
```
Comment (jennypullman) [inline bazel/variables.bzl:2]: Can you add something like: "For chromium-specific flags, see mini_chromium.BUILD."

Comment (jennypullman) [inline bazel/variables.bzl:2]: What do you mean by "The flags are not applied to build third party dependencies."?

Comment (b-maldoca) [reply bazel/variables.bzl]: done

File: bazel/variables.bzl (hunk 5 above)
Comment (b-maldoca) [reply bazel/variables.bzl:5]: the empty line is deliberate. it indicates that the comment is a file wide comment and does not only apply to the next flag (i.e. DEFAULT_MALDOCA_WIN_COPTS)

File: bazel/variables.bzl (hunk 6 above)
Comment (b-maldoca) [reply bazel/variables.bzl]: those flags r only applied to build maldoca/* not maldoca's third party dependencies (i.e. stuff in third_party/* (e.g. chromium)). updated the comment.

File: bazel/variables.bzl (hunk 5 above)
Comment (jennypullman) [reply bazel/variables.bzl:5]: I see, thanks

**Review by jennypullman — APPROVED** [review]

**Comment by b-maldoca** [comment]
thx for the review!

**Pull request merged**